bc_obps/registration/api/user_operator.py

```diff
@@ -52,12 +58,55 @@ def get_user_operator(request, user_operator_id: int):
         ],
     )
 
-    return {
+    user_operator = {
+        **user_operator_fields_dict,
         **user_related_fields_dict,
         "phone_number": user.phone_number.as_e164,  # PhoneNumberField returns a PhoneNumber object and we need a string
         **operator_related_fields_dict,
     }
 
+    return user_operator
+
+
+@router.get("/user-operators", response=List[UserOperatorListOut])
+def list_user_operators(request):
+    qs = UserOperator.objects.all()
+    user_operator_list = []
+
+    for user_operator in qs:
+        user_operator_related_fields_dict = model_to_dict(
+            user_operator,
+            fields=[
+                "id",
+                "status",
+            ],
+        )
+        user = user_operator.user
+        user_related_fields_dict = model_to_dict(
+            user,
+            fields=[
+                "first_name",
+                "last_name",
+                "email",
+            ],
+        )
+        operator = user_operator.operator
+        operator_related_fields_dict = model_to_dict(
+            operator,
+            fields=[
+                "legal_name",
+            ],
+        )
+
+        user_operator_list.append(
+            {
+                **user_operator_related_fields_dict,
+                **user_related_fields_dict,
+                **operator_related_fields_dict,
+            }
+        )
+    return user_operator_list
```

Comment:
If I'm understanding this python code correctly (don't trust me, lol), this code is about transforming the data to give the frontend everything it needs. Probably the same thing could be done with django ninja: https://django-ninja.dev/guides/response/#nested-objects. I don't know which is better, though. I've got another comment about different approaches below